server/src/main/java/io/druid/server/http/DatasourcesResource.java

```diff
@@ -200,6 +201,53 @@ public Response deleteDataSource(
     return Response.ok().build();
   }
 
+  /*
+  Uncomment this method once the method deleteSourceName (just above this) is deleted.
```

Comment:
it should be removed but my point is that we shouldn't remove the endpoint, just deprecate it